server/lib/routes/get-metrics-errors.js

```diff
@@ -0,0 +1,50 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+var logger = require('mozlog')('server.metrics-errors');
+var request = require('request');
+
+var config = require('../configuration');
+var sentryConfig = config.get('sentry');
+
+/**
+ * This module serves as a proxy for front-end errors.
```

Comment:
My primary concern was adding more load to the same process, but this is probably a relatively low volume endpoint and my worry is unfounded.

Carry on.